cmd/mtaBuild.go

```diff
@@ -234,18 +234,33 @@ func runMtaBuild(config mtaBuildOptions,
 		return err
 	}
 
-	log.Entry().Infof("creating build settings information...")
+	log.Entry().Debugf("creating build settings information...")
```

Comment:
as above, possible candidate for a re-use function

Reply:
Fix